src/main/kotlin/Main.kt

```diff
@@ -1,7 +1,7 @@
 import com.elbekD.bot.Bot
 
 fun main(args: Array<String>) {
-    val token = "<TOKEN>"
+    val token = System.getenv("token") ?: "undefined"
     val userName = "<USER_NAME>"
```

Comment:
we have to either add bot name to emvironment aswell or write it here explicitly
second option is ok, because name isn't that big of a secret